.github/workflows/trunk.yml

```diff
@@ -153,6 +153,11 @@ jobs:
         source .ci/scripts/utils.sh
         install_flatc_from_source
         install_executorch
+
+        git config --global user.email "github_executorch@arm.com"
+        git config --global user.name "Github Executorch"
+        bash examples/arm/setup.sh --i-agree-to-the-contained-eula
```

Comment:
Based on the readme file, this might need to be
```
bash examples/arm/setup.sh --i-agree-to-the-contained-eula /opt/arm-sdk
```
to match the arg that's passed to run.sh on line 163.

Or maybe that call shouldn't have `/opt/arm-sdk` in it -- it's weird to have a root-owned directory like that used as a "scratch dir"